packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2318,10 +2318,11 @@ export class WalletService {
   }
 
   _validateAddr(wallet, inaddr, opts) {
```

Comment:
we need to move this to chainService, on a following PR.

Reply:
https://github.com/bitpay/bitcore/pull/2522